CharacterCreation/03_Feats.txt

```diff
@@ -689,113 +696,181 @@ now working with. This feat can only be taken once per weapon.
+	after disembarking. +10 Weapon Reflex at other times.
+36:	+1 Accuracy to ranges R1 and R2 while firing while benefiting from cover 
+	of any kind against your target.
+48:	+10m to range R1, +5m to range R3
+60:	+10% Base damage. For up to one Action per round you may fire your carbine one-handed with  
+	a -2 Accuracy penalty and double any burst or automatic fire Accuracy penalty. Any foregrips or 
+	other attachments requiring the second hand do not provide their benefits during the 
+	Action, but your offhand is free to do something else. If you have taken aim at a target, 
+	you must succeed on a DC 100 Weapons - Carbine check to maintain that aim during and 
+	through the Action.
+
+== Assault/Battle Rifle Expertise ==
+
+0:	+10m to range R2. If this would cause R2 to be greater than R3, 
+	also increase R3 to match the new R2.
+12:	May use Weapons - Assault/Battle Rifle in place of Spotting when 
+	looking through your rifle's scope (warning, takes longer than 
+	wider field-of-view optics).
+24:	Halve the jam chance (applies after additive modifiers, round 
+	down, may be zero) of your assault/battle rifle for the first 3 
+	rounds of its use per combat, once per short rest (i.e. maintenance 
+	& cleaning oportunity).
+36:	+1 Accuracy while both you and your target are benefiting from 
+	cover bonuses against each other.
+48:	+5m to range R1, +10m to range R3.
+60:	+10% Base damage; You may critically hit as normal while firing in 
+	burst mode, and may critically hit with half your normal critical 
+	range while firing in automatic mode.
+
+== Long Rifle Expertise ==
+
+0:	+10% to range R3.
+12:	May use Weapons - Long Rifle (STR) in place of Athletics in order to get to hard-to-reach
+	places, like towers or treetops, that can serve as sniper nests.
+24:	While prone, benefiting from at least light cover, and able to see at least one enemy, 
+	gain an additional +1 cover bonus.
+36:	Gain a stacking +0.5 Accuracy each time you consecutively spend an action shooting a 
+	single target without losing line of sight to it.
+48:	Halve any minimum range associated with your long rifle and attachments, to a max reduction 
+	of 5m.
+60:	+10% Base damage. Increase by 50% any numerical benefits of specialty ammunition you fire 
+	(does not affect Marksman Tech Rounds feats).
+	
+== Heavy Weapon Expertise ==
+
+	Heavy Weapon Expertise does not include Grenade Launchers or Rocket Launchers.
+
+0:	+5m to range R2. If this would cause R2 to be greater than R3, 
+	also increase R3 to match the new R2.
+12:	May use Weapons - Heavy (Cha) in place of Intimidate when out 
+	of combat while brandishing a bigger weapon than the target.
+24:	Heavy weapons no longer recieve the Accuracy penalty associated 
+	with being improvised melee weapons while being used as melee weapons
+	(They still receive a Guard penalty).
+36:	+1 Accuracy against targets when your weapon's APL is at least 1 greater 
+	than the target's armor's APL.

```

Comment:
I'm a little less ecstatic about this one. Why would you be more accurate against heavier armored targets? I have a feeling @rosebrallier would call aesthetic on it. I also notice that there's not a bonus to suppression on here. Could we instead do something like a bonus to accuracy rolls for shots taken in response to an enemy moving into suppression? My fixation on suppression in this case being the assumption that it's what heavy weapons are uniquely good at.

Reply:
Because Accuracy is not only whether your bullet finds the target, but whether the bullet actually does damage instead of careening off their armor in a glancing blow, I thought I could get away with this.
I did consider suppression, but the problem is that for medium and heavy suppression, there aren't any attack rolls involved, only automatic hits.

I have no objections to classifying it as a new issue if we're still not sure about this.

Reply:
Make it so!

Reply:
I agree that we should make this a new issue; it brings up a broader problem of how we collectively want to imagine accuracy and hits.